contents/uix/design-systems.md

```diff
@@ -0,0 +1,130 @@
+<frontmatter>
+  title: Accessibility
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Design Systems
+
+**Author(s): [Tan Heng Yeow](https://github.com/tanhengyeow)**
+
+## What is a Design System
+
+Design Systems has been a trending topic for the past 2 years. The following definition best encapsulates the essence of a Design System.
+> A Design System is the Single Source of Truth which groups all the elements that will allow the teams to design, realize and develop a product. <sub>--[Blog post from UX Collective](https://uxdesign.cc/everything-you-need-to-know-about-design-systems-54b109851969)</sub>
+
+The typical structure of a Design System looks like this:
+<center>
+<img src="https://cdn-images-1.medium.com/max/2600/1*wSHUJh069L618oXqVZYDtA.png" alt="MySQL Workbench GUI Software" width="95%">
+
+_Figure 1. Typical structure of a Design System_ ([source](https://uxdesign.cc/can-design-systems-fix-the-relationship-between-designers-developers-eb12fc9329ab))
+</center>
+
+Below is a brief explanation of the parts that make up a Design System. The explanation includes reference to a living example, [Polaris](https://polaris.shopify.com/), Shopify's Design System.
+- `Rules`: Rules include abstract elements such as brand values, mindset, and shared beliefs. You can refer to Polaris' [product experience principles](https://polaris.shopify.com/patterns-and-guides/product-experience-principles#navigation).
```

Comment:
Consider reproducing one example for each here so that the reader can proceed reading the article without having to look at another page.